### ISSUE TITLE
Possibly outdated LICENSE file?  Artistic-1.0 vs Artistic-1.0-Perl

### ISSUE THREAD

**Issue by swt2c** [houseabsolute/Data-Validate-IP#14]
The LICENSE file says:

"This is free software; you can redistribute it and/or modify it under
the same terms as the Perl 5 programming language system itself.
 
Terms of the Perl programming language system itself
 
a) the GNU General Public License as published by the Free
   Software Foundation; either version 1, or (at your option) any
   later version, or
b) the "Artistic License""

It then provides the content of the "Artistic License" but the content doesn't match that of the current Perl 5 "Artistic License":
https://github.com/Perl/perl5/blob/blead/Artistic

It seems like the LICENSE file may need to be updated to match Perl's, assuming that is the intended license?

**Comment by autarch** [comment]
I think the difference is just the formatting of the license. When there's a future release of this package, the license file will be regenerated and the formatting will be updated.

**Comment by swt2c** [comment]
I don't think it's just formatting.  The `Artistic-1.0-Perl` license has 10 clauses, whereas the one in Data-Validate-IP, which I believe is `Artistic-1.0`, has only 9, for example.

Is the intended license `Artistic-1.0-Perl`?

**Comment by richardfontana** [comment]
BTW "outdated" isn't quite the right term. [Artistic-1.0-Perl](https://spdx.org/licenses/Artistic-1.0-Perl.html) is the *older* version of the license. No one quite knows how Artistic-1.0 came to be, but in my research I have never been able to find any release of Perl that used that license, so it seems to have been introduced by the OSI, perhaps when it first set up the OSI website in the late 1990s.

**Comment by autarch** [comment]
Yeah, I think this is just an older version of the license. It looks like there was an update to the text in a 2023 `Software::License` release. Like I said, it'll be updated if there's a future release. But as it stands, I don't think this will cause any issues.

**Comment by swt2c** [comment]
Sorry to be persistent on this (not worried too much about an update to the license file), but can you please just confirm that this is your intended license for Artistic:
https://spdx.org/licenses/Artistic-1.0-Perl.html

**Comment by autarch** [comment]
Yes, that's the intended license.
